content/en/blog/2022/gc-elections-2022.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Announcing the 2022 OpenTelemetry Governance Committee Election
+linkTitle: OpenTelemetry Governance Committee Election 2022
+date: 2022-09-14
+author: OpenTelemetry Governance Committee
+---
+
+The OpenTelemetry project is excited to announce the 2022 OpenTelemetry
+Governance Committee (GC) election. Nominations are due by end-of-day on 7
+October 2022, with the ratification happening by 8 October 2022 and a list of
+eligible candidates will be shared on 9 October 2022. Voting will take place
+between 18 October 2022 and 20 October 2022, and the final election results will
+be announced 22 October 2022.
+
+## Vote!
+
+If you are a
+[member of standing](https://github.com/open-telemetry/community/blob/main/governance-charter.md#members-of-standing)
+in the OpenTelemetry community, we invite you to participate in this election to
+ensure that the community is well-represented in the Governance Committee. In
+this election four people must be elected, each with two-year terms.
+
+If you have made contributions to our ecosystem not measured by the automatic
+process, you can request
+[an exception](https://github.com/open-telemetry/community/blob/main/governance-charter.md#members-of-standing)
+to participate in the election before 23:59 UTC on 17 October 2022. The voter
+roll with all members of standing and approved exceptions will be published here
+by 11 October 2022 and continuously updated.
+
+Voting will be open between 00:00 UTC on 18 October 2022 00:00 UTC and 20
+October 2022 23:59 UTC on
+[Helios Voting](https://vote.heliosvoting.org/helios/elections/76558134-3384-11ed-8688-02871af94755/view);
+voters will need to sign in with their GitHub account.
+
+To learn more about the election process, read about all of the details
+[here](https://github.com/open-telemetry/community/blob/main/elections/2022/governance-committee-election.md).
+
+## Interested in joining the Governance Committee?
+
+If you’ve been working on OpenTelemetry and seeing it grow, or are an end-user
+that wants to help us make OpenTelemetry better, now’s the time to consider
+running for a seat on the Governance Committee. You can read about the
+Governance Committee's role in
+[this blog](https://medium.com/opentelemetry/opentelemetry-governance-committee-explained-860353baba0)
```

Comment:
Assuming this is going to be referenced in future election announcements as well: Should we move that to blog.opentelemetry.io? (does not have to be part of this PR)

Reply:
Created https://github.com/open-telemetry/opentelemetry.io/issues/1752 to track this.